h/services/bulk_annotation.py

```diff
@@ -101,7 +102,9 @@ def annotation_search(
 
         return [
             BulkAnnotation(
-                username=row.username, authority_provided_id=row.authority_provided_id
+                username=row.username,
+                authority_provided_id=row.authority_provided_id,
+                metadata=row.metadata,
```

Comment:
Adding `row.metadata` to the `BulkAnnotation`'s that `BulkAnnotationService` returns